packages/rs-drive/src/drive/grove_operations/grove_clear/v0/mod.rs

```diff
@@ -16,10 +17,35 @@ impl Drive {
             allow_deleting_subtrees: false,
             trying_to_clear_with_subtrees_returns_error: false,
         };
+
+        let maybe_path_for_logs = if tracing::event_enabled!(Level::TRACE) {
+            Some(path.clone())
+        } else {
+            None
+        };
+
         // we will always return true if there is no error when we don't check for subtrees
-        self.grove
+        let result = self
+            .grove
             .clear_subtree(path, Some(options), transaction)
             .map_err(Error::GroveDB)
-            .map(|_| ())
+            .map(|_| ());
+
+        if tracing::event_enabled!(Level::TRACE) && result.is_ok() {
+            let root_hash = self
+                .grove
+                .root_hash(transaction)
+                .unwrap()
+                .map_err(Error::GroveDB)?;
+
+            tracing::trace!(
+                path = ?maybe_path_for_logs.unwrap().to_vec(),
+                root_hash = ?root_hash,
```

Comment:
maybe `root_hash.to_hex`? (we also need to change this in Drive)

Reply:
The same. Currently, we log them as vectors so it's better to keep the same format to ease comparison. We need to revisit fields for the whole project at once.